tests/wasm/tests.mjs

```diff
@@ -29,13 +29,13 @@ const matchedTransforms = instrumentor.getTransformer(
 assert.ok(matchedTransforms);
 
 const original = await fs.readFile(path.join(import.meta.dirname, './testdata/original.mjs'))
-const output = matchedTransforms.transform(original.toString('utf8'), true);
+const output = matchedTransforms.transform(original.toString('utf8'), 'unknown');
```

Comment:
shouldn't you still keep these tests to assert the boolean directly works?

Reply:
Yes, good point!